packages/pn-personagiuridica-webapp/src/components/Contacts/PecContactItem.tsx

```diff
@@ -156,6 +156,8 @@ const PecContactItem: React.FC = () => {
         pt: defaultSERCQ_SENDAddress ? 0 : 3,
         borderTopLeftRadius: defaultSERCQ_SENDAddress ? 0 : 4,
         borderTopRightRadius: defaultSERCQ_SENDAddress ? 0 : 4,
+        borderBottomLeftRadius: defaultPECAddress?.pecValid ? 0 : 4,
```

Comment:
here you can use value

Reply:
Do you mean 'currentValue'? But here I'm checking if the pec address has been validated because if it's been already validated it needs to be displayed above sercq component and then it do not need to have border radius on the bottom

Reply:
value is empty if pec is not valid